src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -6799,6 +6821,7 @@ subroutine atm_bdy_adjust_scalars_work( scalars_new, scalars_driving, dt, dt_rk,
                cell1 = cellsOnEdge(1,iEdge)
                cell2 = cellsOnEdge(2,iEdge)
 !DIR$ IVDEP
+               !$acc loop collapse(2)
```

Comment:
I guess the compiler will implicitly assume a `vector` here, but wondering if we might need to also explicitly state that here for sake of convention.

Reply:
Thanks for the catch. It's been addressed in the fixup commits pushed today.

Reply:
I noticed there are a few other `!$acc loop collapse(2)` directives. Should we add `vector` to those as well?

Reply:
Thanks for the second catch. @mgduda fixup commit [9d09979](https://github.com/MPAS-Dev/MPAS-Model/pull/1266/commits/9d09979252b3d2510f3f54b94b48a05bfd2baec6) should address this.

Reply:
Thanks!